manifests/server/config.pp

```diff
@@ -100,6 +102,15 @@
     }
   }
 
+  if $manage_postgresql_conf_perms {
+    file { $postgresql_conf_path:
+      ensure => file,
+      owner  => $user,
+      group  => $group,
+      mode   => '0600',
```

Comment:
shouldn't this be using `postgresql_conf_mode`?

Reply:
I think you may have found the thing I was struggling to find. Sometimes you're blind to your own code.